x-pack/filebeat/module/azure/_meta/kibana/7/dashboard/Filebeat-azure-user-activity.json

```diff
@@ -0,0 +1,1311 @@
+                  "order": "desc",
+                  "size": 100,
+                  "type": "terms"
+                },
+                "type": "list"
+              }
+            ],
+            "pinFilters": false,
+            "updateFiltersOnChange": true,
+            "useTimeFilter": false
+          },
+          "title": "User Filters [Filebeat Azure]",
+          "type": "input_control_vis"
+        }
+      },
+      "id": "b0471750-f05b-11e9-90ec-112a988266d5",
+      "migrationVersion": {
+        "visualization": "7.3.1"
+      },
+      "references": [
+        {
+          "id": "filebeat-*",
+          "name": "control_0_index_pattern",
+          "type": "index-pattern"
+        },
+        {
+          "id": "filebeat-*",
+          "name": "control_1_index_pattern",
+          "type": "index-pattern"
+        }
+      ],
+      "type": "visualization",
+      "updated_at": "2019-10-18T09:36:45.050Z",
+      "version": "Wzg0NTcsMV0="
+    },
+    {
+      "attributes": {
+        "description": "",
+        "kibanaSavedObjectMeta": {
+          "searchSourceJSON": {
+            "filter": [],
+            "query": {
+              "language": "kuery",
+              "query": ""
+            }
+          }
+        },
+        "title": "User Activity Overview [Filebeat Azure]",
+        "uiStateJSON": {},
+        "version": 1,
+        "visState": {
+          "aggs": [],
+          "params": {
+            "axis_formatter": "number",
+            "axis_position": "left",
+            "axis_scale": "normal",
+            "default_index_pattern": "metricbeat-*",
+            "default_timefield": "@timestamp",
+            "filter": {
+              "language": "kuery",
+              "query": "event.dataset :\"azure.activitylogs\" and event.category :\"Administrative\" and azure.activitylogs.identity.claims_initiated_by_user.fullname :*"
+            },
+            "id": "61ca57f0-469d-11e7-af02-69e470af7417",
+            "index_pattern": "filebeat-*",
+            "interval": "auto",
+            "isModelInvalid": false,
+            "series": [
+              {
+                "axis_position": "right",
+                "chart_type": "bar",
+                "color": "rgba(1,155,143,1)",
+                "fill": "0.4",
+                "filter": "",
+                "formatter": "number",
+                "hide_in_legend": 0,
+                "id": "61ca57f1-469d-11e7-af02-69e470af7417",
+                "label": "Actions",
+                "line_width": 1,
+                "metrics": [
+                  {
+                    "id": "61ca57f2-469d-11e7-af02-69e470af7417",
+                    "type": "count"
+                  }
+                ],
+                "point_size": 1,
+                "seperate_axis": 0,
+                "split_filters": [
+                  {
+                    "color": "rgba(244,78,59,1)",
+                    "filter": "_exists_:identity.claims.name",
```

Comment:
noticed this filter contains `dentity.claims.name` but that didn't stop the visualization to map the correct claims_initiated_by.. field